akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -255,6 +272,7 @@ private[remote] class SystemMessageDelivery(
       private def clear(): Unit = {
         sendUnacknowledgedToDeadLetters()
         seqNo = 0L // sequence number for the first message will be 1
+        incarnation = outboundContext.associationState.incarnation
```

Comment:
Is incarnation the same as an ActorSystem uuid? If so is there a reason we use a differnt term?

Reply:
Ah understood.